operator/templates/cassandra-env-sh.yaml

```diff
@@ -242,31 +242,29 @@ data:
     # https://blogs.oracle.com/jmxetc/entry/troubleshooting_connection_problems_in_jconsole
     # for more on configuring JMX through firewalls, etc. (Short version:
     # get it working with no firewall first.)
-    #
-    # Cassandra ships with JMX accessible *only* from localhost.
-    # To enable remote JMX connections, uncomment lines below
-    # with authentication and/or ssl enabled. See https://wiki.apache.org/cassandra/JmxSecurity
-    #
-    if [ "x$LOCAL_JMX" = "x" ]; then
-        LOCAL_JMX=yes
-    fi
 
     # Specifies the default port over which Cassandra will be available for
     # JMX connections.
     # For security reasons, you should not expose this port to the internet.  Firewall it if needed.
-    JMX_PORT="7199"
+    JMX_PORT="{{ .Params.JMX_PORT }}"
 
-    if [ "$LOCAL_JMX" = "yes" ]; then
+    {{ if eq .Params.JMX_LOCAL_ONLY "true" }}
       JVM_OPTS="$JVM_OPTS -Dcassandra.jmx.local.port=$JMX_PORT"
       JVM_OPTS="$JVM_OPTS -Dcom.sun.management.jmxremote.authenticate=false"
-    else
+    {{ else }}
       JVM_OPTS="$JVM_OPTS -Dcassandra.jmx.remote.port=$JMX_PORT"
       # if ssl is enabled the same port cannot be used for both jmx and rmi so either
       # pick another value for this property or comment out to use a random port (though see CASSANDRA-7087 for origins)
       JVM_OPTS="$JVM_OPTS -Dcom.sun.management.jmxremote.rmi.port=$JMX_PORT"
 
       # turn on JMX authentication. See below for further options
-      JVM_OPTS="$JVM_OPTS -Dcom.sun.management.jmxremote.authenticate=true"
+      JVM_OPTS="$JVM_OPTS -Dcom.sun.management.jmxremote.authenticate=false"
```

Comment:
Disabling authentication and SSL seems very insecure. This should be configurable and enabled by default.

Reply:
There is an explicit ticket for this. I'd prefer to focus on the actual backup & restore for now, as this might lead to bigger changes in the operator itself. 
I do agree that this should be configurable and be enabled by default though.